.env.test

```diff
@@ -0,0 +1,4 @@
+APP_HOST=localhost
```

Comment:
:axe: This is meant for your personal development only (see above) so doesn't need to be checked in.